geonode_mapstore_client/client/js/components/Menu/Menu.js

```diff
@@ -44,7 +44,7 @@ const Menu = forwardRef(({
                     return (
                         <li key={idx}>
                             <MenuItem
-                                variant={variant}
+                                variant={item.labelId === 'gnhome.createNew' ? 'primary' : variant}
```

Comment:
could we make this configurable instead to rely on the label id that could be modified?

```suggestion
                                variant={item.variant || variant}
```

then add the variant to the localConfig for gnhome.createNew dropdown?